include/circt/Dialect/SV/SVInOutOps.td

```diff
@@ -139,3 +140,38 @@ def ArrayIndexInOutOp
   let assemblyFormat =
     "$input`[`$index`]` attr-dict `:` type($input) `,` type($index)";
 }
+
+def IndexedPartSelectInOutOp : SVOp<"indexed_part_select_inout",
```

Comment:
If the other one gets merged into ArraySliceOp, then it would be good to align the name of this op with that one.